package.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "@calls/common",
+    "version": "0.12.0",
+    "description": "Common code shared between calls webapp, desktop, and mobile.",
+    "license": "MIT",
```

Comment:
Not a bit deal but I'd suggest including the usual LICENSE from other repositories, e.g. https://github.com/mattermost/mattermost-plugin-calls/blob/main/LICENSE